roofit/roofitcore/inc/RooMinimizer.h

```diff
@@ -109,9 +111,7 @@ class RooMinimizer : public TObject {
 
   Int_t       _printLevel ;
   Int_t       _status ;
-  Bool_t      _optConst ;
```

Comment:
I only catch this now, but if you remove or add non-transient member variables to a ROOT class you have to update the class version. Which means that at the end of this file, you have update this:
```C++
ClassDef(RooMinimizer,0) // RooFit interface to ROOT::Fit::Fitter
```
to this
```
ClassDef(RooMinimizer,0) // RooFit interface to ROOT::Fit::Fitter
```

Reply:
Ah good to know about the release cycle point, I indeed intended to bump the number in the final edits. Will do now instead.

Reply:
I see, good that you are keeping track of the class versions :+1: I just prefer to do at at the first edit in the release cycle so we won't forget.